### PR TITLE
Update /containers/lxd page - 18.04

### LINKED ISSUE #2976: Update instructions on /containers/lxd - Build
Check for edits in the copydoc. 
containers/lxd copydoc: https://docs.google.com/document/d/1wBAeklBtadvh4jKuoPFSJxoX2k0zyWt07ZI4nytwYSA/edit#

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Copy updates for `/containers/lxd`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/containers/lxd>
- Make sure page matches [the copy doc](https://docs.google.com/document/d/1wBAeklBtadvh4jKuoPFSJxoX2k0zyWt07ZI4nytwYSA/edit#)

## Issue / Card
Fixes #2976

### CODE REVIEW

**Comment by frankiegranato** [comment]
Hi @steverydz The text I added in brackets is a little large looking. Perhaps a better style would be a 'Note'. See an example of the style used here: https://www.ubuntu.com/download/cloud/try-openstack
p.s. We should probably add UX Review labels to issues like this (even though it is super tiny) because otherwise I am likely to miss them altogether :-)

**Comment by steverydz** [comment]
@frankiegranato OK, I've updated it.

File: templates/containers/lxd.html [hunk 1]
```diff
@@ -122,14 +122,21 @@ <h2>Getting started with LXD</h2>
...
             <div class="p-code-snippet">
               <input class="p-code-snippet__input" value="sudo lxd init" readonly="readonly">
               <button class="p-code-snippet__action">Copy to clipboard</button>
             </div>
             <p style="margin-top: 1rem;">You can generally hit Enter to accept the default answers to the questions, and once you&rsquo;re setup you can start to launch machines.</p>
+
+            <div class="p-notification--caution">
+              <div class="p-notification__response">
+                <p><span class="p-notification__status">Note:</span></p>
+                <p>(LXD is on Ubuntu 16.04 LTS Server or greater by default. On Ubuntu Desktop LXD will need to be installed)</p>
```
Comment (pmahnke) [inline templates/containers/lxd.html]: @steverydz remove the '(' and ')' no parens...

**Comment by steverydz** [comment]
@pmahnke Done

**Pull request merged**